component/Page/Profile/index.tsx

```diff
@@ -102,7 +103,7 @@ export default function ProfilePage(props: PageProps) {
     }
 
     const getInputMessageContainerTopPosition = () => {
-        return getContainerHeight() + window.scrollY - 47 - 24
+        return getContainerHeight() + window.scrollY - 96 - 24
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `getInputMessageContainerTopPosition` function modifies the offset from the top of the window. This could affect the layout of your page depending on the rest of your CSS. Make sure that this new value doesn't cause any unintended layout issues.
```diff
- return getContainerHeight() + window.scrollY - 96 - 24
+ return getContainerHeight() + window.scrollY - 47 - 24
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->